src/core/js/helpers.js

```diff
@@ -1,416 +1,411 @@
-     * {{/all}}
-     */
-    all: function() {
-      var args = Array.prototype.slice.call(arguments, 0, -1);
-      var block = Array.prototype.slice.call(arguments, -1)[0];
-      return _.all(args) ? (block.fn ? block.fn(this) : true) : (block.inverse ? block.inverse(this) : false);
-    },
-
-    /**
-     * Equivalent to:
-     *  if (!conditionA && !conditionB)
-     * @example
-     * {{#none displayTitle body instruction}}
-     * <div class='component__header {{_component}}__header'></div>
-     * {{/none}}
-     */
-    none: function() {
-      var args = Array.prototype.slice.call(arguments, 0, -1);
-      var block = Array.prototype.slice.call(arguments, -1)[0];
-
-      return !_.any(args) ? (block.fn ? block.fn(this) : true) : (block.inverse ? block.inverse(this) : false);
-    },
-
-    /**
-     * Allow JSON to be a template i.e. you can use handlebars {{expressions}} within your JSON
-     */
-    compile: function(template, context) {
-      if (!template) {
-        return '';
-      }
-      if (template instanceof Object) template = template.toString();
-      var data = this;
-      if (context) {
-        // choose between a passed argument context or the default handlebars helper context
-        data = (!context.data || !context.data.root ? context : context.data.root);
-      }
-      return Handlebars.compile(template)(data);
-    },
-
-    /**
-     * Allow JSON to be a template and accessible text
-     */
-    compile_a11y_text: function(template, context) {
-      Adapt.a11y.log.deprecated('a11y_text is no longer required. https://tink.uk/understanding-screen-reader-interaction-modes/');
-      return helpers.compile.call(this, template, context);
-    },
-
-    /**
-     * Allow JSON to be a template and normalized text
-     */
-    compile_a11y_normalize: function(template, context) {
-      if (!template) {
-        return '';
-      }
-      if (template instanceof Object) template = template.toString();
-      return Handlebars.helpers.a11y_normalize.call(this, helpers.compile.call(this, template, context));
-    },
-
-    /**
-     * Remove all html tags except styling tags
-     */
-    compile_a11y_remove_breaks: function(template, context) {
-      if (!template) {
-        return '';
-      }
-      return Handlebars.helpers.a11y_remove_breaks.call(this, helpers.compile.call(this, template, context));
-    },
-
-    /**
-     * makes the _globals object in course.json available to a template
-     */
-    import_globals: function(context) {
-      if (context.data.root._globals) {
-        return '';
-      }
-      context.data.root._globals = Adapt.course.get('_globals');
+import Adapt from 'core/js/adapt';
+
+const defaultAriaLevels = {
+  '_menu': 1,
+  '_menuGroup': 2,
+  '_menuItem': 2,
+  '_page': 1,
+  '_article': 2,
+  '_block': 3,
+  '_component': 4,
+  '_componentItem': 5,
+  '_notify': 1
+};
+
+const helpers = {
+
+  lowercase: function(text) {
```

Comment:
could possibly put all these helper functions into a Helpers class?

Reply:
It would make it somewhat difficult to do the `Handlebars.registerHelper(name, helpers[name]);` bit at the bottom because the functions wouldn't be enumerable on the instance.